src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -313,12 +318,13 @@ public static string GetInsertionText(CompletionItem item)
             string filterText = null,
             SupportedPlatformData supportedPlatforms = null,
             ImmutableDictionary<string, string> properties = null,
-            ImmutableArray<string> tags = default)
+            ImmutableArray<string> tags = default,
+            bool isComplexTextEdit = false)
```

Comment:
i'm not seeing any root calls tot hese methods.  do we need this parameter at all?